release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -24,16 +24,13 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 You must use {op-system} machines for the control plane, and you can use either {op-system} or {op-system-base} for compute machines.
 //Removed the note per https://issues.redhat.com/browse/GRPA-3517
 
-For {product-title} 4.12 on `x86_64` architecture, Red Hat has added a 6-month Extended Update Support (EUS) phase that extends the total available lifecycle from 18 months to 24 months. For {product-title} 4.12 running on 64-bit ARM (`aarch64`), {ibm-power-name} (`ppc64le`), and {ibm-z-name} (`s390x`) architectures, the EUS lifecycle remains at 18 months.
+//Even-numbered release lifecycle verbiage
+Starting from {product-title} 4.14, the Extended Update Support (EUS) phase for even-numbered releases increases the total available lifecycle to 24 months on all supported architectures, including `x86_64`, 64-bit ARM (`aarch64`), {ibm-power-name} (`ppc64le`), and {ibm-z-name} (`s390x`) architectures.
+Beyond this, Red Hat also offers a 12-month additional EUS add-on, denoted as Additional EUS Term 2, that extends the total available lifecycle from 24 months to 36 months. The Additional EUS Term 2 is also available on all architecture variants of {product-title}.
 
-Starting with {product-title} 4.14, each EUS phase for even numbered releases on all supported architectures, including `x86_64`, 64-bit ARM (`aarch64`), {ibm-power-name} (`ppc64le`), and {ibm-z-name} (`s390x`) architectures, has a total available lifecycle of 24 months.
-
-Starting with {product-title} 4.14, Red Hat offers a 12-month additional EUS add-on, denoted as _Additional EUS Term 2_, that extends the total available lifecycle from 24 months to 36 months. The Additional EUS Term 2 is available on all architecture variants of {product-title}.
-
-For more information about this support, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
-
-//TODO: The line below should be used when it is next appropriate. Revisit in August 2023 time frame.
-Maintenance support ends for version 4.12 on 25 January 2025 and goes to extended life phase. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
+//Odd-numbered release lifecycle verbiage
+//For {product-title} {product-version}, the total available lifecycle is 16 months on all supported architectures, including `x86_64`, 64-bit ARM (`aarch64`), {ibm-power-name} (`ppc64le`), and {ibm-z-name} (`s390x`) architectures. Even-numbered releases have Extended Update Support (EUS) phases that extend the available lifecycle to 24 or 36 months.
```

Comment:
The lifecycle of non-eus versions seems to be 18 months.

Reply:
```suggestion
//Starting from {product-title} 4.12/4.14, the EUS phase for odd-numbered relese, such as {product-title} 4.15, on all supported architectures, including `x86_64`, 64-bit ARM (`aarch64`), {ibm-power-name} (`ppc64le`), and {ibm-z-name} (`s390x`) architectures is 18 months.
```

Reply:
For OpenShift Container Platform 4.12 on x86_64 architecture, the EUS phase extends from  18 months to 24 months?

Reply:
Hi @shellyyang1989 . The following statement for the add-on is still correct?

> Starting from OpenShift Container Platform 4.14, Red Hat offers a 12-month additional EUS add-on, denoted as Additional EUS Term 2, that extends the total available lifecycle from 24 months to 36 months. The Additional EUS Term 2 is available on all architecture variants of OpenShift Container Platform.

Reply:
>For OpenShift Container Platform 4.12 onwards fro odd-numbered releases on x86_64 architecture, the EUS phase extends from 18 months to 24 months?

For odd-numbered releases, they are not EUS releases, the lifecycle of them is 18 months.

Reply:
>The following statement for the add-on is still correct?

Yes, it's correct.

Reply:
- commencing with 4.8, all **even** numbered minor release are donated EUS releases (odd-numbered minor releases are not EUS releases)

- commencing with 4.12, **a six-month EUS phase** is applied to EUS releases, the total lifecycle is 24 months. EUS is provided with x86 on 4.12

- commencing with 4.14, EUS is provided with all arches

- commencing with 4.14, **an additional EUS Term 2 of 12 months** is provided with all arches, the total lifecycle is 36 months